src/cmd/build.js

```diff
@@ -131,7 +131,8 @@ async function defaultPackageCreator({
   await eventToPromise(stream, 'close');
 
   if (showReadyMessage) {
-    log.info(`Your web extension is ready: ${extensionPath}`);
```

Comment:
Adding a note where? In the docs? I saw the [warning you added](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Getting_started_with_web-ext#Packaging_your_extension) to the docs already. It might scare some people into using an ID when they don't have to but I guess it's helpful.